src/main/java/io/quarkus/search/app/indexing/IndexingService.java

```diff
@@ -163,29 +163,35 @@ private void createIndexes() {
 
     private void indexAll() {
         Log.info("Indexing...");
-        try (Rollover rollover = Rollover.start(searchMapping);
-                Closer<IOException> closer = new Closer<>()) {
-            // Reset the database before we start
-            clearDatabaseWithoutIndexes();
+        try (FailureCollector failureCollector = new FailureCollector(indexingConfig.errorReporting().enabled(),
+                indexingConfig.errorReporting().gitRepository(), indexingConfig.errorReporting().gitIssueId())) {
```

Comment:
This is nitpicking, but you'd save yourself some trouble by just passing `indexingConfig.errorReporting()`:

```suggestion
        try (FailureCollector failureCollector = new FailureCollector(indexingConfig.errorReporting())) {
```